crates/store/src/server/api.rs

```diff
@@ -607,6 +630,18 @@ fn invalid_argument<E: core::fmt::Display>(err: E) -> Status {
     Status::invalid_argument(err.to_string())
 }
 
+#[instrument(target = COMPONENT, skip_all, err)]
+fn validate_account_ids(
+    account_ids: &[generated::account::AccountId],
+) -> Result<Vec<AccountId>, Status> {
+    account_ids
+        .iter()
+        .cloned()
+        .map(AccountId::try_from)
+        .collect::<Result<_, ConversionError>>()
+        .map_err(|_| invalid_argument("Digest field is not in the modulus range"))
```

Comment:
I think I am also at fault here for copying this function but forgetting to change this `map_err()` error message.

Reply:
Missed this while reviewing draft